### PR TITLE
Cross reference GitHub issues in the specification

### PR DESCRIPTION
**Pull request by mfoltzgoogle**

### CODE REVIEW

File: Overview.src.html [hunk 1]
```diff
@@ -1460,16 +1467,22 @@ <h3 id="use-cases">
       TODO
     </h3>
     <p class="open-issue">
-      Need algorithm for receive message
+      <a href="https://github.com/w3c/presentation-api/issues/34">ISSUE 34:
+      Flesh out the algorithms</a>
     </p>
+    <h2 class="no-num">
+      Security Considerations
```
Comment (anssiko) [inline Overview.src.html]: We should make this a single section called "Security and privacy considerations" (and remove its own "Privacy Considerations" section).

File: index.html [hunk 2]
```diff
@@ -411,8 +421,8 @@ <h4 id="video-and-image-sharing"><span class="secno">1.1.2 </span>
     <p>
       <b>Requirements:</b> R1, R3, R4, R5, R7
     </p>
-    <h4 id="gaming"><span class="secno">1.1.3 </span>
-      Gaming
+    <h4 id="multiplayer-gaming"><span class="secno">1.1.3 </span>
+      Multiplayer Gaming
```
Comment (anssiko) [inline index.html]: We should use sentence-case for consistency, i.e. "Multiplayer gaming".

File: Overview.src.html [hunk 3]
```diff
@@ -316,7 +320,7 @@ <h3 id="use-cases">
       <b>Requirements:</b> R1, R3, R4, R5, R7
     </p>
     <h4>
-      Gaming
+      Multiplayer Gaming
```
Comment (anssiko) [inline Overview.src.html]: We should use sentence-case for consistency, i.e. "Multiplayer gaming".

**Comment by anssiko** [comment]
@mfoltzgoogle This looks good to me with minor comments. As a stylistic issue, I suggest dropping "Open issue:" prefix defined in CSS for the selector p.open-issue:before.

@drott Please review and merge.

All - HTML preview with @mfoltzgoogle's changes at: http://rawgit.com/mfoltzgoogle/presentation-api/open-issues/index.html

**Comment by drott** [comment]
LGTM, thanks!

**Pull request merged**